### PR TITLE
Fix dropdown prop, allow composer styling

### PR DESCRIPTION
**Pull request by thanhsonng**
- refactor: Add proptypes docs & clean up Dropdown.Container
- fix: Only use compatible props for Container
- refactor: Organize props destructuring assignments
- refactor: Format Composer file
- feat: Allow custom styling for Composer from props

### CODE REVIEW

File: src/components/Composer/index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (namdaoduy) [inline src/components/Composer/index.js:1]: @thanhsonng Why should we disable this rule?

Comment (thanhsonng) [reply src/components/Composer/index.js]: @namdaoduy This rule causes linting error in these files and thus prevents new commit to be made.
I just pushed a new commit that disables this rule globally, as per your recommendation.

File: .eslintrc [hunk 2]
```diff
@@ -8,45 +8,57 @@
...
+      1,
+      {
+        "extensions": [
+          ".js",
+          ".jsx"
+        ]
+      }
+    ],
+    "react/jsx-no-bind": 0,
+    "react/jsx-props-no-spreading": 0,
```
Comment (thanhsonng) [inline .eslintrc:55]: Besides sorting eslint rules, I also added this new global rule: 
```
"react/jsx-props-no-spreading": 0
```

**Review by namdaoduy — APPROVED** [review]

**Comment by namdaoduy** [comment]
Please update CHANGELOG.md @thanhsonng

**Comment by namdaoduy** [comment]
Thanks @thanhsonng for your fine work 💯

**Comment by thanhsonng** [comment]
my pleasure

**Pull request merged**